src/ngamsServer/ngamsServer/ngamsFileUtils.py

```diff
@@ -154,8 +180,13 @@ def lookup_partner_site_file_status(ngas_server,
             logger.info(genLog("NGAMS_INFO_FILE_AVAIL", [file_id, partner_address]))
             disk_info = status_info.getDiskStatusList()[0]
             file_info = disk_info.getFileObjList()[0]
-            host = partner_address
-            port = partner_port
+            # This is a bit of a hack because the host ID may not contain
+            # the fully qualified address. We append the domain name from
+            # the partner site address. This should work because they are
+            # part of the same cluster.
+            host, domain, port = parse_host_id(disk_info.getHostId())
+            if domain is None:
```

Comment:
You need to check here if `partner_domain` for `None`, this is what's causing the tests to fail.

Reply:
Yes. You are correct. I have resolved this.